modules/exploits/windows/local/lenovo_systemupdate.rb

```diff
@@ -0,0 +1,193 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class Metasploit3 < Msf::Exploit::Local
+  Rank = ExcellentRanking
```

Comment:
I'm not the ranking master at all, this wiki page is the master: https://github.com/rapid7/metasploit-framework/wiki/Exploit-Ranking =)

According to the "oracle" I guess it's suitable for ExcellentRanking :)